source/features/esc-to-deselect-line.tsx

```diff
@@ -12,7 +13,7 @@ function listener(event: KeyboardEvent): void {
 	if (
 		event.key === 'Escape' && // Catch `Esc` key
 		isLineSelected() &&
-		!(event.target instanceof HTMLTextAreaElement || event.target instanceof HTMLInputElement) // If a field isn’t focused
+		(event.target && !isEditable(event.target)) // If a field isn’t focused
```

Comment:
`event.target` could be null

Reply:
Target _can_ be null. You can change the function to accept a null parameter and drop the checks outside the function